src/test/parse-fail/issue-46186.rs

```diff
@@ -0,0 +1,18 @@
+// Copyright 2017 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+// compile-flags: -Z parse-only
+
+struct Struct {
+    a: usize,
+}; //~ ERROR expected item, found ';'
+//~| NOTE consider removing the semicolon
+
+fn main() {}
```

Comment:
Could you turn this into a `ui` test?

Reply:
Should I go ahead and remove this file from parse-fail then?